profile_library/amazon/SK705DI/model.json

```diff
@@ -0,0 +1,34 @@
+{
+  "calculation_enabled_condition": "{{ is_state('[[entity]]', 'playing') }}",
+  "calculation_strategy": "linear",
+  "created_at": "2024-07-03T17:22:35.339969",
+  "device_type": "smart_speaker",
+  "linear_config": {
+    "calibrate": [
+      "10 -> 3.09",
+      "20 -> 3.09",
+      "30 -> 3.11",
+      "40 -> 3.11",
+      "50 -> 3.11",
+      "60 -> 3.13",
+      "70 -> 3.15",
+      "80 -> 3.14",
+      "90 -> 3.25",
+      "100 -> 3.51",
+      "0 -> 3.06"
+    ]
+  },
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly Plus Plug S",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 1,
+    "SLEEP_TIME": 2,
+    "VERSION": "v1.12.6:docker"
+  },
+  "name": "Amazon Echo (1st Generation)",
+  "standby_power": 3.1,
```

Comment:
Let's set this to 3.06 so it's not higher than the measured power when muted (volume 0)